spec/README.md

```diff
@@ -17,27 +114,23 @@ To create a course, its provider and any other objects needed to create a valid
 course and save it to the database (obeying nullability and foreign key rules):
 
 ```
-let(:course)  { create :course }
+def when_a_course_exists  { create :course }
```

Comment:
This would need to be assigned to an instance variable. Also, I think we should distinguish between feature tests and unit/request specs.

Reply:
Same comment applies to the relevant changes below.